docs/concepts/vega-protocol.md

```diff
@@ -2,69 +2,273 @@
 
-Tokenholders can create proposals using the APIs.
+For proposals to change market parameters, there are additional requirements. The network will also calculate:
+* The LP participation rate, which is the sum of the equity like share of all LPs who cast a vote - `LP participation rate = SUM (equity like share of all LPs who cast a vote)`
+* The rate of 'for' votes cast by liquidity providers, calculated as the sum of all who voted 'for', divided by the LP participation rate - `LP for rate = SUM (all who voted for) / LP participation rate`
 
-<!-- 
 :::info
-Vote on proposals on the [Vega token governance page](https://token.vega.xyz/governance).
+Vote on active proposals on the [Vega token dApp](https://token.fairground.wtf/governance).
 :::
 
-## Collateral management
-The collateral for restricted mainnet is exclusively VEGA, which is used for staking and rewards. 
- 
+#### 5. Enacting changes
+If a proposal receives enough token weight in favour within the enactment period, the change/addition is accepted (except for a freeform proposal), and will be enacted on the enactment date defined in the proposal. If a proposal doesn't require an enactment date and time, then it will be enacted immediately.
+
+Note the enactment date must be at least the minimum enactment period for the proposal type/subtype (specified by a network parameter) after voting closes.
+
+## Asset governance
+Details on asset governance to come. 
+
+<!--
+### New asset proposal [WIP]
+-->
+
+## Market governance
+Markets are proposed and voted into existence by Vega tokenholders. The parameters for a market all need to be defined in the proposal.
+
+Some market parameters can also be changed. They can only be proposed by a liquidity provider with enough equity-like stake in the market, and need to be voted for by a sufficient number of tokenholders and/or liquidity providers.
+
+When creating a market governance proposal, whether it is for a new market or to change parameters for an existing market, it's recommended that you sense check the proposal and share the final details with the tokenholder community before proposing, so that you can garner support and make any necessary amends. 
+
+Read more:
+* [Vega community forum](https://community.vega.xyz): Share your draft proposals for community discussion.
+* [New market proposal](../tutorials/proposals/new-market-proposal.md): Guide to submitting a proposal for a new market using the command line
+* [Update market proposal](../tutorials/proposals/update-market-proposal.md): Guide to submitting a proposal to change a market using the command line
+
+### Propose a new market
+Tokenholders can propose new markets, which then need to be voted on by other tokenholders. If the market proposal gets a majority of tokeholder support, then it will be enacted. The required majority is defined by the network parameter `governance.proposal.market.requiredMajority`. 
```

Comment:
I think there are similar net params for market governance, there are also various other net params that could be referenced for a number of the comments mentioned on this page. Should we include their details?

Reply:
Bonus points for adding a component that shows the value of a network param